content/influxdb/clustered/install/deploy.md

```diff
@@ -18,6 +56,25 @@ kubectl apply \
   --namespace influxdb
 ```
 
+<!-------------------------------- END kubectl -------------------------------->
+{{% /tab-content %}}
+{{% tab-content %}}
+<!-------------------------------- BEGIN kubit -------------------------------->
+
+1.  [Install the `kubit` operator](https://github.com/kubecfg/kubit/#installation)
+  and related tools.
+
+2.  Use the `kubit local apply` command to apply your custom-configured
+    `myinfluxdb.yml` and deploy your InfluxDB Cluster.
+
+    ```sh
+    kubit local apply myinfuxdb.yml
+    ```
```

Comment:
Step 1 here requires the elevated permissions to install the `AppInstance` CRD. Once you run the 
`kubectl apply -k ...` part, that means the operator is running in the target cluster and there is no need to run `kubit local apply` (because it is already running).

I believe step 2 has been confused with the "I have minimal permissions/I'd like to act as the operator and try things out" possibility in `kubit`. If step 1 has been done, then a customer should run `kubectl apply -f myinfluxdb.yml`, i.e. "apply my `AppInstance` custom resource that I have" [1] and let the `kubit` operator handle that resource in the way it has been programmed to do.

By running the `kubit local apply` step, it would circumvent the previously installed operator entirely, as under the hood the `local apply` means "untar the given OCI bundle/package (Clustered), populate the required values, render the manifests as the operator would and then apply them". If a customer wants `kubit` to handle all of that then they should pass the `AppInstance` for Clustered to the Kubernetes API server (`kubectl apply`) and allow `kubit` to do the rest of it once it sees that the object has been created.

I understand that this can be quite confusing though, so I'm happy to go over it in more detail/step through it if you like too :blush: 

---
1. _Technically_ this does work, but only if you are wanting to act as the operator to test things out and step 1 isn't required in that case. I think it is potentially confusing to customers that they would _require_ `kubit` in order to apply their `AppInstance` (`myinfluxdb.yml` file), when in fact it is simply the regular `kubectl apply` operation that they would be used to.